lib/dashboard/console.js

```diff
@@ -38,22 +38,30 @@ class Console {
     var pluginCmds = this.plugins.getPluginsProperty('console', 'console');
     for (let pluginCmd of pluginCmds) {
       let pluginOutput = pluginCmd.call(this, cmd, {});
-      if (pluginOutput !== false && pluginOutput !== 'false') return callback(pluginOutput);
+      //console.log("plugin output: ", pluginOutput)
+      if (pluginOutput !== false && pluginOutput !== 'false') 
+        return callback(pluginOutput);
+      else if (pluginOutput === undefined) 
+        return;
```

Comment:
wouldn't this just hang the console?

it probably should be something like pluginOutput !==undefined in the if above

Reply:
it didn't hang the console for me.